rllib/execution/parallel_requests.py

```diff
@@ -0,0 +1,150 @@
+import logging
+from typing import Any, Callable, Dict, List, Optional, TYPE_CHECKING
+
+import ray
+from ray.actor import ActorHandle
+from ray.rllib.utils.annotations import ExperimentalAPI
+
+if TYPE_CHECKING:
+    from ray.rllib.agents.trainer import Trainer
+
+logger = logging.getLogger(__name__)
+
+
+@ExperimentalAPI
+def asynchronous_parallel_requests(
```

Comment:
this function looks really familiar. are we not replacing some existing logics with this util func call somewhere?

Reply:
Yeah, sorry, moved it into a new module for better clarity: This function may not only be used to collect SampleBatches from a RolloutWorker, but works generically on any set (and types!) of ray remote actors.

Reply:
I removed the old code (not used anywhere anyways) in `replay_ops.py`.